composer.json

```diff
@@ -13,11 +13,16 @@
       "Rakit\\Validation\\": "src"
     }
   },
+  "autoload-dev": {
+    "psr-4": {
+      "Rakit\\Validation\\Tests\\": "tests"
+    }
+  },
   "require": {
     "php": ">=5.5.0",
     "ext-mbstring": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "4.*"
+    "phpunit/phpunit": "^4.8 || ^5.7 || ^6.5"
```

Comment:
Is there any reason for this? I think you should just bump it to `>=6` ...

Reply:
@adelowo, you're right. I just set the `^6.5` version to support the `php-7.0+` versions :).